retriever/engines/postgres.py

```diff
@@ -258,13 +258,19 @@ def get_connection(self):
         import psycopg2 as dbapi
 
         self.get_input()
-        conn = dbapi.connect(
+        try:
+        	conn = dbapi.connect(
```

Comment:
Indentation seems a bit off at some places.

Reply:
Fixed it